### ISSUE TITLE
Sealing water gives 50 salt 

### ISSUE THREAD

**Issue by Arsakes** [Vintage-Civ/bug-reports#6]
Despite previous attempts to nerf the salt production sealing 50l of water in barrel gives 50 untis of salt.

The tooltip was changed though (it says 5 units of salt will be produced)

**Comment by edendour** [comment]
Yeah, it worked at first but now it's giving 50 salt per barrel of water.
